tutorial/creating_packages/create_your_first_package.rst

```diff
@@ -313,6 +313,9 @@ located in the user home folder under the ``.conan2`` folder. Conan will use the
 You already used the :command:`conan list` command to list the recipes and binaries stored
 in the local cache. 
 
+An **important** note: the Conan cache is read-only. Do not modify, add, remove or change
```

Comment:
Wording suggestion: 
"the contents of the Conan cache are private to the Conan client - modifying, adding, removing or changing files inside the Conan cache is undefined behaviour likely to cause breakages."

(pedantic: it is "read-only" from the users point of view, but obviously not for the filesystem, in case users think filesystem)

Reply:
Makes sense, changing it.